### DISCUSSION TITLE
prisma-plugin: getting error when adding plugin to builder

### DISCUSSION THREAD

**Discussion by nerdroid23** [hayes/pothos#420] [Q&A] [answered]
Hello, I want to preface that I am a complete noob, I apologize if I am not doing something correctly.

I am trying to use the prisma-plugin and i am getting an odd error and I have not yet figured out what it wrong.  I am using nextjs and graphql-yoga. This is the error that I am getting

<img width="703" alt="image" src="https://user-images.githubusercontent.com/21242932/173284115-55d4051e-8afd-4b10-bf8f-3c60288db38a.png">

```js
// builder.ts
import SchemaBuilder from "@pothos/core";
import { prisma } from "../prisma/prisma";
import { PrismaPlugin } from "@pothos/plugin-prisma";

import type PrismaTypes from "@pothos/plugin-prisma/generated";

export const builder = new SchemaBuilder<{
  PrismaTypes: PrismaTypes;
}>({
  plugins: [PrismaPlugin],
  prisma: {
    client: prisma,
  },
});
```

```js
// prisma.ts
import { PrismaClient } from "@prisma/client";

declare global {
  // allow global `var` declarations
  // eslint-disable-next-line no-var
  var prisma: PrismaClient | undefined;
}

export const prisma =
  global.prisma ||
  new PrismaClient({
    log: ["query"],
  });

if (process.env.NODE_ENV !== "production") global.prisma = prisma;
```

Any help would be greatly appreciated! 🙏🏾

**Comment by hayes** [comment, marked as answer]
You want to use the default export of the plugin, not the named class export.  

Eg import PrismaPlugin instead of import { PrismaPlugin }

**Reply by nerdroid23** [reply]
Thanks! I don't know how I missed this 🤦🏾‍♂️
